src/main/drivers/compass/compass_ak8963.c

```diff
@@ -213,90 +231,227 @@ static bool ak8963Read(int16_t *magData)
                 if (retry) {
```

Comment:
As I said before, this jump is always taken. Your previous version of code did honor `retry`, which may have caused lockups ...

Reply:
I prefer to not touch this for now. The state engine need some review again to remove last statics. This PR changes getting much more impact already now then getting the AK8963 SPI driver working.